dependencies.props

```diff
@@ -56,6 +58,10 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)coreclr/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <RemoteDependencyBuildInfo Include="MicrosoftNETCoreApp">
```

Comment:
The `RemoteDependencyBuildInfo`s actually are one per repo, so this should stay as `CoreSetup` (same for the CurrentRef).